src/renderer/components/+cluster-settings/components/cluster-accessible-namespaces.tsx

```diff
@@ -16,7 +16,7 @@ export class ClusterAccessibleNamespaces extends React.Component<Props> {
   render() {
     return (
       <>
-        <SubTitle title="Accessible Namespaces" />
+        <SubTitle title="Accessible Namespaces" id="accessible-namespaces" />
```

Comment:
Btw, you can highlight element with CSS when `#accessible-namespaces` in the URL (easier to spot the block after scrolling).

_Example:_

![image](https://user-images.githubusercontent.com/6377066/108518268-0acefa00-72d1-11eb-98a7-c914ef758eea.png)

Reply:
Ah excellent, great solution. I had thought that this would be a good idea but didn't know about this solution (I thought it would be a large hassle so I didn't try) 👍